.github/workflows/publish_website.yml

```diff
@@ -44,30 +44,25 @@ jobs:
         with:
           python-version: 3.x
           cache: "pip"
-      - id: "gcp-auth"
-        name: "Authenticating to Google Cloud"
-        uses: "google-github-actions/auth@v1"
-        with:
-          token_format: "access_token"
-          credentials_json: "${{ secrets.IREE_OSS_GITHUB_RUNNER_BASIC_TRUST_SERVICE_ACCOUNT_KEY }}"
-          create_credentials_file: false
       - name: Installing dependencies
         run: |
           pip install -r docs/website/requirements.txt
           pip install requests
+          sudo apt update
+          sudo apt install -y ninja-build
+          echo "CC=clang" >> $GITHUB_ENV
+          echo "CXX=clang++" >> $GITHUB_ENV
       - name: Generating release index
         run: |
           ./build_tools/scripts/generate_release_index.py \
             --repo="${GITHUB_REPOSITORY}" \
             --output=docs/website/docs/pip-release-links.html
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1.2
```

Comment:
Should we pin that on a specific hash?

Reply:
We could switch all of our workflows back to hashes instead of versions. Would want to make that change consistently to all workflows.